### ISSUE TITLE
Ricochet doesn't run on Windows Server 2012

### ISSUE THREAD

**Issue by special** [ricochet-im/ricochet#166]
A user reports that they're not able to run Ricochet 1.1.0 on several different Windows Server 2012 installations, both VM and native. There is an error at startup from libGLESv2.dll, with the exception code 0xc000001d, which seems to translate to "illegal instruction". The exception offset is 0x0009f4df.

No other reports of this so far.

**Comment by special** [comment]
I'm unable to reproduce this with Windows Server 2012 under VMware, with or without graphics acceleration enabled.

**Comment by special** [comment]
There is a second report of this, with windows 8.1.

It was fixed in one case by installing OEM video drivers for the Intel GPU.

**Comment by Spydar007** [comment]
Just tried it, getting the same.

When opening Ricochet on Windows Server 2012 R2, it "stops working".

The "problem signature" is as follows: 
```
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	ricochet.exe
  Application Version:	1.1.2.0
  Application Timestamp:	56bd53b1
  Fault Module Name:	libGLESv2.dll
  Fault Module Version:	0.0.0.0
  Fault Module Timestamp:	56bd48fc
  Exception Code:	c000001d
  Exception Offset:	000a137c
  OS Version:	6.3.9600.2.0.0.272.7
  Locale ID:	1033
  Additional Information 1:	5861
  Additional Information 2:	5861822e1919d7c014bbb064c64908b2
  Additional Information 3:	1318
  Additional Information 4:	13181ae637ac2592fac0ca2dd662cdcc
```

**Comment by special** [comment]
Should be fixed by #367
